justfile

```diff
@@ -12,15 +12,27 @@ default:
 
 # Cargo build everything.
 build:
-  cargo build --workspace --all-targets --all-features
+  cargo build -p corepc-client --all-targets --all-features
+  cargo build -p corepc-types --all-targets --all-features
+  cargo build -p jsonrpc --all-targets --all-features
+  cargo build -p corepc-node --all-targets --features=29_0
+  (cd verify && cargo build --all-targets)
 
 # Cargo check everything.
 check:
-  cargo check --workspace --all-targets --all-features
+  cargo check -p corepc-client --all-targets --all-features
+  cargo check -p corepc-types --all-targets --all-features
+  cargo check -p jsonrpc --all-targets --all-features
+  cargo check -p corepc-node --all-targets --features=29_0
+  (cd verify && cargo check --all-targets)
```

Comment:
We could use `cargo check --manifest-patch verify/Cargo.toml` instead to save cd'ing into `verify`.